development/idp-local/config/config.php

```diff
@@ -806,20 +802,13 @@
      * ],
      *
      */
-    'memcache_store.servers' => [
-        [
-            [
-                'hostname' => $MEMCACHE_HOST1,
-                'port' => $MEMCACHE_HOST1_PORT,
-            ],
-        ],
-        [
-            [
-                'hostname' => $MEMCACHE_HOST2,
-                'port' => $MEMCACHE_HOST2_PORT,
-            ],
-        ],
-    ],
+//    'memcache_store.servers' => [
```

Comment:
Why not /* */ comment around the outdated code?

Reply:
I think doing that would make it too similar to comments detailing what config values do.

Reply:
Not to mention that PhpStorm doesn't use /* */ when you press Ctrl-/